annotation/core/models.py

```diff
@@ -43,26 +58,32 @@ class Annotation(models.Model):
     """A human annotation of a prompt-continuation pair."""
     timestamp = models.DateTimeField(auto_now=True, null=True)
     annotator = models.ForeignKey(User, on_delete=models.DO_NOTHING)
-    text = models.ForeignKey(EvaluationText, on_delete=models.DO_NOTHING)
+    generation = models.ForeignKey(Generation, on_delete=models.DO_NOTHING)
     boundary = models.IntegerField()
-    tags = models.ManyToManyField(Tag, related_name="annotation_tags")
-    revision = models.TextField()
     points = models.IntegerField()
+    revision = models.TextField()
     attention_check = models.BooleanField(default=False)
 
     def __str__(self):
         return self.annotator.username + " " + str(self.timestamp)
 
 
-class Group(models.Model):
+class Playlist(models.Model):
     """A grouping of several prompt-continuation pairs."""
     name = models.CharField(max_length=128, blank=True)
     description = models.TextField(blank=True)
```

Comment:
Could you add a second TextField here: detailed_description or some similar name. For each PlayList we have the layman's text i.e. "Look at these super cool news articles." and the technical description i.e. "Annotate continuations from GROVER on the New York Times Corpus 2003-2013."

Reply:
Added a `long_description` field to be used for full technical information.